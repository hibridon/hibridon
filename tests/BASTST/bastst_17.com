inp=bastst_17.inp
out=bastst_17.out
run
save=save_17.inp