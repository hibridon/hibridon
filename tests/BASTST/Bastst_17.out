      BASTST                                     
 DUMMY POTENTIAL FOR TESTING PURPOSES            
 ==============================
     *** Parameters:
 JTOT1  =  20        JTOT2  =  20        JTOTD  =   5        JLPAR  =   1
 NERG   =   1        NUMAX  =   0        NUMIN  =   0        NUD    =   1
 LSCREEN=  48        IPRINT =   1
 FSTFAC =  15.00     RINCR  =  1.000     RCUT   =  30.00     RENDAI =  1.000    
 RENDLD =  1.000     RSTART =  1.000     SPAC   = 0.1000     TOLAI  =  1.000    
 XMU    =  16.47    
     *** CH2X system parameters:
 NTERM  =   4        NUMPOT =   0        IPOTSY =   2        IOP    =   1
 IVBEND =   3        JMAX   =  15
 EMAX   =  1500.    
 LAMMIN:    0   1   2   3
 LAMMAX:    6   5   6   5
 MPROJ:     0   1   2   3
     *** Flags:
 AIRYFL= F    BASTST= T    BATCH = F    CHLIST= T    CSFLAG= F    FLAGHF= F
 FLAGSU= F    IHOMO = F    IPOS  = F    LOGDFL= F    NOPRIN= F    NUCROS= F
 PHOTOF= F    PRAIRY= F    PRLOGD= F    PRPART= F    PRSMAT= F    PRT2  = F
 PRXSEC= F    READPT= F    RSFLAG= F    T2TEST= F    TWOMOL= F    WAVEFL= F
 WRPART= F    WRSMAT= F    WRXSEC= F    BOUNDC= F
 ** Energies:     5000.000000
 NOUT: 13; JOUT:  0   1   2   3   4   5   6   7   8   9  10  11  12
 INDOUT:    0   1  -1   2  -2   3  -3   4  -4
 ==============================

 --------------------------------------------------------------------------
           HIBRIDON SCATTERING CODE V 5.00

      AUTHORS: M. ALEXANDER, D. MANOLOPOULOS, H.-J. WERNER, B. FOLLMEG,
               P. DAGDIGIAN
 CONTRIBUTORS: D. LEMOINE, P. VOHRALIK, G. COREY, R. JOHNSON, T. ORLIKOWSKI,
          A. BERNING, A. DEGLI-ESPOSTI, C. RIST, B. POUILLY, J. KLOS, Q. MA,
          G. VAN DER SANDEN, M. YANG, F. DE WEERD, S. GREGURICK, F. LIQUE
 --------------------------------------------------------------------------
           BUILD CONFIGURATION:
CMAKE BUILD DATE: 2023-03-19 at 06:05
CMAKE BUILD TYPE: Debug
CODE GIT REVISION: 2d62711070dd74d49874476e89aa8d5b1cd9a5dd
SYS: Darwin 22.2.0 x86_64
COMPILER: /opt/intel/oneapi/compiler/2022.2.1/mac/bin/intel64/ifort
COMPILE OPTIONS: 
 --------------------------------------------------------------------------
            CURRENT HARDWARE CONFIGURATION:
 
      Model Name: MacBook Pro
      Model Identifier: MacBookPro14,3
      Processor Name: Quad-Core Intel Core i7
      Processor Speed: 2.9 GHz
      Number of Processors: 1
      Total Number of Cores: 4
      L2 Cache (per Core): 256 KB
      L3 Cache: 8 MB
      Hyper-Threading Technology: Enabled
      Memory: 16 GB
 --------------------------------------------------------------------------

 -----------------------------------------------------------------------------
 All publications resulting from use of the integrators included in the
 Hibridon code must include the following reference:

 D. E. Manolopoulos, J. Chem. Phys. 85, 6425 (1986); M. H. Alexander and D. E.
 Manolopoulos, J. Chem. Phys. 80, 2044 (1987).

 All publications involving the determination ofphotodissociation cross sections
 must also include the following reference:

 M. H. Alexander, Comput. Phys. Commun, 75, 87 (1993).

 All publications investigating flux redistribution must include the following references:

 M. H. Alexander, J. Chem. Phys. 95, 8931 (1991); D. E. Manolopoulos and
 M. H. Alexander, J. Chem. Phys.  97, 2527 (1992).

 All publications resulting from use of the Hibridon package must include
 the following reference:

 HIBRIDON is a package of programs for the time-independent quantum treatment
 of inelastic collisions and photodissociation written by M. H. Alexander,
 D. E.  Manolopoulos, H.-J. Werner, and B. Follmeg, with contributions by
 P. F. Vohralik, D. Lemoine, G. Corey, B. Johnson, T. Orlikowski, A. Berning,
 A. Degli-Esposti, C. Rist, P. Dagdigian, B. Pouilly, G. van der Sanden, M. Yang, F. de Weerd, S. Gregurick, J. Klos, and F. Lique
 -----------------------------------------------------------------------------

 ** LABEL:          BASTST                                     
 ** POT NAME:  DUMMY POTENTIAL FOR TESTING PURPOSES            
 ** DATE:      Mar 19 06:06:51 2023

 *** WARNING:  NAIRY .NE. 1 BUT AIRYFL .EQ. .FALSE.
 ** NMAX= 500  NAIRY= 500
 ** TOTAL NUMBER OF ANISOTROPIC TERMS IN POTENTIAL = 12

 **  CH2(X 3B1) (0,V_BEND,0) VIBRONIC LEVEL **
     RMU=  16.4700    V_BEND=  0
     E=5000.00  JTOT=  20   JLPAR= 1

 OPEN CHANNELS ELIMINATED WHICH ARE CLOSED AT R=   30.00

  CHANNEL LIST SORTED BY ENERGY
   #   N  IS   L   EINT(CM-1)
   1   1   0  20    15.648
   2   2   0  19    46.960
   3   2   0  21    46.960
   4   3   0  18    93.967
   5   3   0  20    93.967
   6   3   0  22    93.967
   7   4   0  17   156.717
   8   4   0  19   156.717
   9   4   0  21   156.717
  10   4   0  23   156.717
  11   5   0  16   235.272
  12   5   0  18   235.272
  13   5   0  20   235.272
  14   5   0  22   235.272
  15   5   0  24   235.272
  16   6   0  15   329.706
  17   6   0  17   329.706
  18   6   0  19   329.706
  19   6   0  21   329.706
  20   6   0  23   329.706
  21   6   0  25   329.706
  22   7   0  14   440.105
  23   7   0  16   440.105
  24   7   0  18   440.105
  25   7   0  20   440.105
  26   7   0  22   440.105
  27   7   0  24   440.105
  28   7   0  26   440.105
  29   1   1  19   523.489
  30   1   1  21   523.489
  31   1  -1  20   530.702
  32   2  -1  19   560.421
  33   2  -1  21   560.421
  34   2   1  18   565.777
  35   2   1  20   565.777
  36   2   1  22   565.777
  37   8   0  13   566.558
  38   8   0  15   566.558
  39   8   0  17   566.558
  40   8   0  19   566.558
  41   8   0  21   566.558
  42   8   0  23   566.558
  43   8   0  25   566.558
  44   8   0  27   566.558
  45   3  -1  18   605.004
  46   3  -1  20   605.004
  47   3  -1  22   605.004
  48   3   1  17   615.703
  49   3   1  19   615.703
  50   3   1  21   615.703
  51   3   1  23   615.703
  52   4  -1  17   664.460
  53   4  -1  19   664.460
  54   4  -1  21   664.460
  55   4  -1  23   664.460
  56   4   1  16   682.258
  57   4   1  18   682.258
  58   4   1  20   682.258
  59   4   1  22   682.258
  60   4   1  24   682.258
  61   9   0  12   709.151
  62   9   0  14   709.151
  63   9   0  16   709.151
  64   9   0  18   709.151
  65   9   0  20   709.151
  66   9   0  22   709.151
  67   9   0  24   709.151
  68   9   0  26   709.151
  69   9   0  28   709.151
  70   5  -1  16   738.800
  71   5  -1  18   738.800
  72   5  -1  20   738.800
  73   5  -1  22   738.800
  74   5  -1  24   738.800
  75   5   1  15   765.429
  76   5   1  17   765.429
  77   5   1  19   765.429
  78   5   1  21   765.429
  79   5   1  23   765.429
  80   5   1  25   765.429
  81   6  -1  15   828.042
  82   6  -1  17   828.042
  83   6  -1  19   828.042
  84   6  -1  21   828.042
  85   6  -1  23   828.042
  86   6  -1  25   828.042
  87   6   1  14   865.196
  88   6   1  16   865.196
  89   6   1  18   865.196
  90   6   1  20   865.196
  91   6   1  22   865.196
  92   6   1  24   865.196
  93   6   1  26   865.196
  94  10   0  11   867.961
  95  10   0  13   867.961
  96  10   0  15   867.961
  97  10   0  17   867.961
  98  10   0  19   867.961
  99  10   0  21   867.961
 100  10   0  23   867.961
 101  10   0  25   867.961
 102  10   0  27   867.961
 103  10   0  29   867.961
 104   7  -1  14   932.212
 105   7  -1  16   932.212
 106   7  -1  18   932.212
 107   7  -1  20   932.212
 108   7  -1  22   932.212
 109   7  -1  24   932.212
 110   7  -1  26   932.212
 111   7   1  13   981.533
 112   7   1  15   981.533
 113   7   1  17   981.533
 114   7   1  19   981.533
 115   7   1  21   981.533
 116   7   1  23   981.533
 117   7   1  25   981.533
 118   7   1  27   981.533
 119  11   0  10  1043.039
 120  11   0  12  1043.039
 121  11   0  14  1043.039
 122  11   0  16  1043.039
 123  11   0  18  1043.039
 124  11   0  20  1043.039
 125  11   0  22  1043.039
 126  11   0  24  1043.039
 127  11   0  26  1043.039
 128  11   0  28  1043.039
 129  11   0  30  1043.039
 130   8  -1  13  1051.344
 131   8  -1  15  1051.344
 132   8  -1  17  1051.344
 133   8  -1  19  1051.344
 134   8  -1  21  1051.344
 135   8  -1  23  1051.344
 136   8  -1  25  1051.344
 137   8  -1  27  1051.344
 138   2   2  19  1068.013
 139   2   2  21  1068.013
 140   2  -2  18  1068.077
 141   2  -2  20  1068.077
 142   2  -2  22  1068.077
 143   8   1  12  1114.402
 144   8   1  14  1114.402
 145   8   1  16  1114.402
 146   8   1  18  1114.402
 147   8   1  20  1114.402
 148   8   1  22  1114.402
 149   8   1  24  1114.402
 150   8   1  26  1114.402
 151   8   1  28  1114.402
 152   3   2  18  1115.297
 153   3   2  20  1115.297
 154   3   2  22  1115.297
 155   3  -2  17  1115.618
 156   3  -2  19  1115.618
 157   3  -2  21  1115.618
 158   3  -2  23  1115.618
 159   4   2  17  1178.040
 160   4   2  19  1178.040
 161   4   2  21  1178.040
 162   4   2  23  1178.040
 163   4  -2  16  1178.997
 164   4  -2  18  1178.997
 165   4  -2  20  1178.997
 166   4  -2  22  1178.997
 167   4  -2  24  1178.997
 168   9  -1  12  1185.487
 169   9  -1  14  1185.487
 170   9  -1  16  1185.487
 171   9  -1  18  1185.487
 172   9  -1  20  1185.487
 173   9  -1  22  1185.487
 174   9  -1  24  1185.487
 175   9  -1  26  1185.487
 176   9  -1  28  1185.487
 177  12   0   9  1234.410
 178  12   0  11  1234.410
 179  12   0  13  1234.410
 180  12   0  15  1234.410
 181  12   0  17  1234.410
 182  12   0  19  1234.410
 183  12   0  21  1234.410
 184  12   0  23  1234.410
 185  12   0  25  1234.410
 186  12   0  27  1234.410
 187  12   0  29  1234.410
 188  12   0  31  1234.410
 189   5   2  16  1255.998
 190   5   2  18  1255.998
 191   5   2  20  1255.998
 192   5   2  22  1255.998
 193   5   2  24  1255.998
 194   5  -2  15  1258.205
 195   5  -2  17  1258.205
 196   5  -2  19  1258.205
 197   5  -2  21  1258.205
 198   5  -2  23  1258.205
 199   5  -2  25  1258.205
 200   9   1  11  1263.747
 201   9   1  13  1263.747
 202   9   1  15  1263.747
 203   9   1  17  1263.747
 204   9   1  19  1263.747
 205   9   1  21  1263.747
 206   9   1  23  1263.747
 207   9   1  25  1263.747
 208   9   1  27  1263.747
 209   9   1  29  1263.747
 210  10  -1  11  1334.701
 211  10  -1  13  1334.701
 212  10  -1  15  1334.701
 213  10  -1  17  1334.701
 214  10  -1  19  1334.701
 215  10  -1  21  1334.701
 216  10  -1  23  1334.701
 217  10  -1  25  1334.701
 218  10  -1  27  1334.701
 219  10  -1  29  1334.701
 220   6   2  15  1348.896
 221   6   2  17  1348.896
 222   6   2  19  1348.896
 223   6   2  21  1348.896
 224   6   2  23  1348.896
 225   6   2  25  1348.896
 226   6  -2  14  1353.232
 227   6  -2  16  1353.232
 228   6  -2  18  1353.232
 229   6  -2  20  1353.232
 230   6  -2  22  1353.232
 231   6  -2  24  1353.232
 232   6  -2  26  1353.232
 233  10   1  10  1429.482
 234  10   1  12  1429.482
 235  10   1  14  1429.482
 236  10   1  16  1429.482
 237  10   1  18  1429.482
 238  10   1  20  1429.482
 239  10   1  22  1429.482
 240  10   1  24  1429.482
 241  10   1  26  1429.482
 242  10   1  28  1429.482
 243  10   1  30  1429.482
 244  13   0   8  1442.055
 245  13   0  10  1442.055
 246  13   0  12  1442.055
 247  13   0  14  1442.055
 248  13   0  16  1442.055
 249  13   0  18  1442.055
 250  13   0  20  1442.055
 251  13   0  22  1442.055
 252  13   0  24  1442.055
 253  13   0  26  1442.055
 254  13   0  28  1442.055
 255  13   0  30  1442.055
 256  13   0  32  1442.055
 257   7   2  14  1456.453
 258   7   2  16  1456.453
 259   7   2  18  1456.453
 260   7   2  20  1456.453
 261   7   2  22  1456.453
 262   7   2  24  1456.453
 263   7   2  26  1456.453
 264   7  -2  13  1464.066
 265   7  -2  15  1464.066
 266   7  -2  17  1464.066
 267   7  -2  19  1464.066
 268   7  -2  21  1464.066
 269   7  -2  23  1464.066
 270   7  -2  25  1464.066
 271   7  -2  27  1464.066
 272  11  -1  10  1499.066
 273  11  -1  12  1499.066
 274  11  -1  14  1499.066
 275  11  -1  16  1499.066
 276  11  -1  18  1499.066
 277  11  -1  20  1499.066
 278  11  -1  22  1499.066
 279  11  -1  24  1499.066
 280  11  -1  26  1499.066
 281  11  -1  28  1499.066
 282  11  -1  30  1499.066
ILAM=  1 LAMNUM(ILAM) =    282
ILAM=  2 LAMNUM(ILAM) =   1502
ILAM=  3 LAMNUM(ILAM) =   3179
ILAM=  4 LAMNUM(ILAM) =   4657
ILAM=  5 LAMNUM(ILAM) =    676
ILAM=  6 LAMNUM(ILAM) =   2590
ILAM=  7 LAMNUM(ILAM) =   4848
ILAM=  8 LAMNUM(ILAM) =   1163
ILAM=  9 LAMNUM(ILAM) =   2677
ILAM= 10 LAMNUM(ILAM) =   4224
ILAM= 11 LAMNUM(ILAM) =   1268
ILAM= 12 LAMNUM(ILAM) =   2275
 ** TOTAL NUMBER OF NONZERO V2 MATRIX ELEMENTS IS 29341
 ** BASTST=.TRUE.; TEST OF SUBROUTINE BASIS
 **** END OF CALCULATION ****
      MAXIMUM NUMBER OF CHANNELS USED WAS:     0
      TIMING:  ELAPSED 00:00:00 / CPU 00:00:00 / MP RATIO   14.7 %
      CURRENT DATE:  Mar 19 06:06:51 2023
 ===============================================================================
